src/types/Todo.ts

```diff
@@ -1,5 +1,5 @@
 export interface Todo {
-  id: number;
+  id?: number;
```

Comment:
how can this field be optional? I mean in all cases even when you show temporary todo we'll have an id